dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -170,14 +170,21 @@ RUN mkdir /licenses
 COPY --from=home {{ $beatHome }}/LICENSE.txt /licenses
 COPY --from=home {{ $beatHome }}/NOTICE.txt /licenses
 
-{{- if contains .image_name "-cloud" }}
+{{- if or (eq .Variant "cloud") (eq .Variant "service") }}
 COPY --from=home /opt /opt
 # Generate folder for a stub command that will be overwritten at runtime
 RUN mkdir /app && \
     chown {{ .user }}:{{ .user }} /app
 {{- end }}
 
-{{- if (and (contains .image_name "-complete") (contains .from "ubuntu"))  }}
+{{- if eq .Variant "service" }}
+RUN apk add --no-cache python-3.12 py3.12-pip && \
+    pip install {{ $beatHome }}/data/service_downloads/connectors-*.zip && \
```

Comment:
Would be safer to unzip and  `make install`, as that'll also give it a python virtual environment, and keep it resileint to any other system python deps that might end up in this image one day.

Reply:
I can do that, seems a little un-needed being this is the only python used in the image. The simple `pip install` without requiring unzip to be installed in the image seems better to me.

Reply:
it's best practice (AFAIU) to not use system python if you can help it. Also by using a `make` target, it allows us to hook into other setup bits. For example, our `make isntall` insures that `bin/pip install --upgrade pip` and `bin/pip install --upgrade setuptools` are both run, and also ensure that the right `requirements/$ARCH.txt` is selected, depending upon the OS that it's being run on.